sphinx/themes/basic/layout.html

```diff
@@ -150,20 +150,24 @@ <h3>{{ _('Navigation') }}</h3>
 {%- block extrahead %} {% endblock %}
   </head>
   {%- block body_tag %}<body>{% endblock %}
+
+{%- block skiplink %}
+  <a id="skiplink" href="#main">Skip to main content</a>
+{% endblock %}
+
 {%- block header %}{% endblock %}
 
 {%- block relbar1 %}{{ relbar() }}{% endblock %}
 
 {%- block content %}
   {%- block sidebar1 %} {# possible location for sidebar #} {% endblock %}
-
     <div class="document">
   {%- block document %}
       <div class="documentwrapper">
       {%- if render_sidebar %}
         <div class="bodywrapper">
       {%- endif %}
-          <div class="body" role="main">
+          <div class="body" role="main" id="main">
```

Comment:
For this theme and a few others it's necessary to duplicate the node ID because they cleanly inherit the HTML from `basic`.